src/main/java/nextstep/subway/path/domain/Payment.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.subway.path.domain;
+
+import lombok.Getter;
+import nextstep.subway.common.SubwayErrorMessage;
+import nextstep.subway.exception.IllegalDistanceValueException;
+
+import java.util.Set;
+
+@Getter
+public class Payment {
+
+    private static final Set<PaymentPolicy> policies = Set.of(
+            new DefaultPaymentPolicy(),
+            new UnderFifthPaymentPolicy(),
+            new OverFifthPaymentPolicy()
+    );
+
+    private final int payment;
+
+    public Payment(long distance) {
+        this.payment = setPayment(distance);
+    }
```

Comment:
개인적으로 생성자에 멤버 변수가 아닌 다른 필드 값을 넣으면 추후에 헷갈릴 우려가 있어 지양하는 편인데요, 
아래와 같이 해볼 수 있을 것 같은데 진묵님 생각은 어때요?
```suggestion
    private Payment(long payment) {
        this.payment = setPayment(payment);
    }
        
     public static Payment of(long distance) {
         return this.Payment(calculate(distance));
     }
    
```

Reply:
피드백 감사합니다!!
사실 저도 정적 메서드를 이용해서 객체 생성을 자주 하는 편인데, 저는 기존에는 생성할 때 의미가 있는 이름을 사용하기 위한 목적으로 주로 사용했습니다.
이번에 리뷰어님께서 말씀해주신 서로 다른 필드 값을 넣을때 사용하는것도 굉장히 좋다고 느껴서 수정했습니다!!
애매하게 있던 저의 기준이 좀 더 명확하게 정리된것 같아요!!

좋은 피드백 주셔서 감사합니다~~